### DISCUSSION TITLE
Config option to disable claiming of naturally generated strucutres.

### DISCUSSION THREAD

**Discussion by vijaypondini** [GriefPrevention/GriefPrevention#1422] [Ideas]
Can we add a config to disable claiming of structures in the game?
disable_structure_claims: true
it would disable claiming things like naturally generated ocean monuments, woodland mansions, fortresses, pillager outposts etc.

Would prevent people from claiming entire fortresses to themselves and blocking access, etc.

**Comment by Jikoo** [comment]
There's no way to detect a structure generating using the Bukkit API without rewriting worldgen. The best you could automatically do would be evaluate blocks during claim, which can be done by an addon.
